webpack.config.js

```diff
@@ -41,6 +41,7 @@ const config = {
     'tippy.js',
     'uuid',
     'v-tooltip',
+    'vue',
```

Comment:
We do not need to set Vue as external here if we do not have it as a dependency.

Reply:
hmm.. I doublecheck that. But I needed it for some reason...

Reply:
> hmm.. I doublecheck that. But I needed it for some reason...

you moved vue into the `dependencies` block... maybe it was included in the build, then?

Reply:
That was my thought too. But I want to be sure.

Reply:
tested. Looks like You where right 9d449c0814abc8dd85d62a77a470a10f2fc1dbaa

Reply:
ok. Got it. 
We need it. 
We don't need it to build demosplan-ui, but we need it to tell the using app, that we need vue.
We wouldn't need that if we would have it in the `dependencies` section. BUt as `devDependency` it get stripped out during the build and we have to set it as `external`. And thats what we want - imho.

Reply:
> Shouldn't it be a peerDependency?

thats what i think, too, overthinking it once more.